### DISCUSSION TITLE
Remove tag

### DISCUSSION THREAD

**Discussion by DevBlokJochem** [Minestom/Minestom#1113] [Help] [answered]
How can I remove a tag?
![image](https://user-images.githubusercontent.com/71212639/169708325-71f8b25d-d551-43ab-8469-549b85edf4f9.png)
these are the only options I can use.

**Comment by TheMode** [comment, marked as answer]
Set the tag to null.
```java
withTag(tag, null)
```

**Reply by DevBlokJochem** [reply]
oke
